.ci/register_package.jl

```diff
@@ -70,4 +70,11 @@ mktempdir() do download_dir
     BinaryBuilder.upload_to_github_releases(repo, tag, download_dir; verbose=verbose)
 end
 BinaryBuilder.push_jll_package(name, build_version)
+mktempdir() do dir
+    Pkg.activate(dir)
+    pkg = PackageSpec(; url = "https://github.com/$(repo)")
+    Pkg.add(pkg)
```

Comment:
I know that the package is already in the local depot, I'm just not sure how to find it, so I'm trying to install from the remote repository